modules/ocf_ssh/manifests/makeservices.pp

```diff
@@ -7,7 +7,7 @@
   file {
     '/opt/share/makeservices':
       ensure => directory,
-      mode   => '0700',
+      mode   => '0500',
```

Comment:
It shouldn't be. I made `/opt/share/enforcer` write-only (comments on that?) and I suppose committed it to this branch instead of another by mistake.